sdk/mixedreality/mixedreality-authentication/swagger/README.md

```diff
@@ -0,0 +1,26 @@
+# Azure Mixed Reality Authentication Protocol Layer
+
+> see [https://aka.ms/autorest](https://aka.ms/autorest)
+
+Run `rushx generate:client` or `update.ps1` to generate code.
+
+## Configuration
+
+```yaml
+package-name: "@azure/mixedreality-authentication"
+title: MixedRealityStsRestClient
+description: Mixed Reality Authentication client
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../
+source-code-folder-path: ./src/generated
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/aa19725fe79aea2a9dc580f3c66f77f89cc34563/specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2019-02-28-preview/mr-sts.json
+add-credentials: false
+use-extension:
+  "@autorest/typescript": "6.0.0-dev.20200727.1"
```

Comment:
Perhaps use a more recent one?, a lot changed since then. Also use `hide-clients: true` to help with hiding generated code from the documentation.
```suggestion
  "@autorest/typescript": "6.0.0-dev.20210121.2"
hide-clients: true
```

Reply:
Interesting. The `hide-clients: true` seems to tag the clients with an `@hidden` tag. So many ways to hide things.

Reply:
That was the status que before. This will be fixed to insert `@internal` soon. Feel free to do that update by hand for now.

Reply:
I prefer to avoid hand editing the generated code, but I'm happy to update the tooling when it's ready.